modules/cli/src/main/scala/coursier/cli/bootstrap/BootstrapSpecificOptions.scala

```diff
@@ -2,63 +2,82 @@ package coursier.cli.bootstrap
 
 import caseapp.{ExtraName => Short, HelpMessage => Help, ValueDescription => Value, _}
 import coursier.install.RawAppDescriptor
+import coursier.cli.options.OptionGroup
 
 // format: off
 final case class BootstrapSpecificOptions(
+  @Group(OptionGroup.bootstrap)
   @Short("o")
     output: Option[String] = None,
+  @Group(OptionGroup.bootstrap)
+  @Hidden
   @Short("f")
     force: Boolean = false,
+  @Group(OptionGroup.bootstrap)
+  @Hidden
```

Comment:
We shouldn't hide this either. It's a normal feature of the `bootstrap` command. Maybe it could just be put in a dedicated group, like "Bootstrap format", only displayed at the end in the help messages.

Reply:
I am okay to show it in the `Bootstrap` option group.